### PR TITLE
Update r-markdown to 1.4

### PR DESCRIPTION
**Pull request by mdehollander**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-rmarkdown/meta.yaml [hunk 1]
```diff
@@ -16,17 +16,17 @@ build:
...
 requirements:
   build:
-    - r
+    - r-base
     - r-catools
-    - r-htmltools >=0.2.4
-    - r-knitr >=1.12.3
+    - r-htmltools >=0.3.5
+    - r-knitr >=1.14
     - r-yaml >=2.1.5
```
Comment (dyusuf) [inline recipes/r-rmarkdown/meta.yaml]: r-rmarkdown 1.4 needs more dependencies as it is shown in the travis output:

`ERROR: dependencies ‘base64enc’, ‘jsonlite’, ‘rprojroot’ are not available for package ‘rmarkdown’`

`conda skeleton` is handy for building new cran pkg, which captures all needed dependencies, as in this case:

`conda skeleton cran rmarkdown`

the resulting recipe skeleton is almost ready for the publication while just needing to add the hash.

Comment (mdehollander) [inline recipes/r-rmarkdown/meta.yaml]: @dyusuf Thanks for the skeleton suggestion, did not know about that. I updated the meta.yaml file according to the suggestions of the skeleton file.

**Comment by mdehollander** [comment]
The linux build is now successful but on osx there is this error:
```
The following specifications were found to be in conflict:
  - r-base 3.3.1*
  - r-knitr >=1.14 -> r-base 3.3.2*
  - r-knitr >=1.14 -> r-evaluate >=0.10
```

I will look into that later this week. If anyone else knows how to fix it, feel free :)

**Comment by daler** [comment]
@mdehollander `r-rmarkdown` is in the `r` channel but only up to version 1.3. (https://anaconda.org/r/r-rmarkdown/files). It's such a general package that it should not be maintained in bioconda.

@johanneskoester is the `conda-forge` R migration ready for adding something like this?

**Comment by johanneskoester** [comment]
Yes, it can be added. Everything is ready.

**Comment by mdehollander** [comment]
What can I do? Does this package need to be moved to `conda-forge`? Tell me because I understand that bioconda is not the place to have this package, but where is best is uncleat to me.

**Comment by bgruening** [comment]
@mdehollander yes conda-forge would be the place to go.

**Comment by rvalieris** [comment]
this package is now on conda-forge, you can submit a PR here: https://github.com/conda-forge/r-markdown-feedstock

**Pull request closed without merging**